x11-terms/xterm/xterm-372.ebuild

```diff
@@ -38,6 +38,10 @@ RDEPEND="${DEPEND}
 
 DOCS=( README{,.i18n} ctlseqs.txt )
 
+PATCHES=(
```

Comment:
Needs a revbump as it affects runtime behaviour (copy the ebuild to -r1 and ekeyword ~all it.